Makefile

```diff
@@ -11,6 +11,7 @@ GC_DARWIN_AMD64=${GC_VERSION}-darwin-amd64
 GC_FREEBSD_AMD64=${GC_VERSION}-freebsd-amd64
 GC_LINUX_AMD64=${GC_VERSION}-linux-amd64
 GC_WINDOWS_AMD64=${GC_VERSION}-windows-amd64
+WINDOWS_BIN=${BIN}.exe
```

Comment:
I would have declared WINDOWS_BIN right after BIN.
But it is a detail, otherwise LGTM

Reply:
Fixed in d8d2d66